src/Raven.Studio/typescript/models/wizard/serverSetup.ts

```diff
@@ -105,7 +105,7 @@ class serverSetup {
                 const portPart = this.unsecureSetup().port() || '8080';
                 return "http://" + this.unsecureSetup().ips()[0].ip() + ':' + portPart; 
             case "LetsEncrypt":
-                return "https://a." + this.domain().domain() + ".dbs.local.ravendb.net" + this.getPortPart();
+                return "https://a-" + this.domain().domain() + ".dbs.local.ravendb.net" + this.getPortPart();
```

Comment:
Can we get this from the server, rather than hard code the '-` or `.` choice?

Reply:
we will (reported in separate issue) - for now I've changed this to make it work.